controllers/sriovnetwork_controller.go

```diff
@@ -141,6 +146,14 @@ func (r *SriovNetworkReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	err = r.Get(ctx, types.NamespacedName{Name: netAttDef.Name, Namespace: netAttDef.Namespace}, found)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			targetNamespace := &corev1.Namespace{}
+			err = r.Get(ctx, types.NamespacedName{Name: netAttDef.Namespace}, targetNamespace)
+			if errors.IsNotFound(err) {
+				// Target namespace doesn't exist, avoid retrying with exponential backoff
```

Comment:
nit: perhaps reword this one 
> // Target namespace doesn't exist, avoid retrying with exponential backoff

// Target namespace doesn't exist. nothing to do. reconcile will be re-triggered when namespace is created.

or just remove comment and put info in log like:

reqLogger.Info("Target namespace doesn't exist, NetworkAttachmentDefinition will be created when namespace is available", "Namespace", netAttDef.Namespace, "Name", netAttDef.Name)

Reply:
going for the latter